.github/workflows/test.yml

```diff
@@ -80,6 +80,13 @@ jobs:
         run: |
           bundle exec fastlane check
 
+      - name: Upload test reports on failure
+        uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
+        if: failure()
+        with:
+          name: test-reports
+          path: app/build/reports/tests/
```

Comment:
We should remove this, right?

Reply:
It'll be removed when the branch updates 👍